### PR TITLE
Community feature for slf4j-simple-logger 

### LINKED ISSUE #1654: Considering adding a community feature for slf4j-simple-logger
https://github.com/GoodforGod/slf4j-simple-logger

### PR DESCRIPTION
**Pull request by wetted**
https://github.com/GoodforGod/slf4j-simple-logger

closes #1654

### CODE REVIEW

File: starter-core/src/main/java/io/micronaut/starter/feature/logging/Slf4jSimpleLogger.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+@Singleton
+public class Slf4jSimpleLogger implements CommunityFeature {
+    public static final String NAME = "goodforgod-slf4j-simple-logger";
+
+    private static final String SLF4J_SIMPLE_LOGGER_ARTIFACT_ID = "slf4j-simple-logger";
+
+    private static final Dependency SLF4J_SIMPLE_LOGGER_DEPENDENCY = Dependency.builder()
+            .lookupArtifactId(SLF4J_SIMPLE_LOGGER_ARTIFACT_ID)
+            .compile()
```
Comment (sdelamo) [inline starter-core/src/main/java/io/micronaut/starter/feature/logging/Slf4jSimpleLogger.java]: This should be `runtimeOnly`

**Review by sdelamo — CHANGES_REQUESTED** [review]
Ping @GoodforGod

**Review by sdelamo — CHANGES_REQUESTED** [review]
Thinking about this. I think we should create a OneOf marker interface for Slf4j implementation. Users should not add this dependency and logback at the same time. They should add either this one or logback.

**Comment by wetted** [comment]
> Thinking about this. I think we should create a OneOf marker interface for Slf4j implementation. Users should not add this dependency and logback at the same time. They should add either this one or logback.

@sdelamo I already changed the `Slf4jSimpleLogger` feature to be a `LoggingFeature`. I believe that satisfies what you want since the other logger features are `LoggingFeature`, and that is a `OneOfFeature`.

Or did I misunderstand?

File: starter-api/src/test/groovy/io/micronaut/starter/api/FeatureControllerSpec.groovy [hunk 2]
```diff
@@ -41,7 +41,7 @@ class FeatureControllerSpec extends Specification {
                 'agorapulse-micronaut-permissions',
                 'agorapulse-micronaut-slack',
                 'agorapulse-micronaut-worker',
-                'camunda-zeebe',
```
Comment (sdelamo) [inline starter-api/src/test/groovy/io/micronaut/starter/api/FeatureControllerSpec.groovy:44]: trailing comma is also allowed in groovy lists

**Review by sdelamo — APPROVED** [review]

**Pull request merged**